string robot_id
geometry_msgs/Pose pose